libs/ngrx-toolkit/src/lib/mutation/rx-mutation.ts

```diff
@@ -10,8 +10,8 @@ import {
   tap,
 } from 'rxjs';
 
-import { concatOp, FlatteningOperator } from './flattening-operator';
-import { Mutation, MutationResult, MutationStatus } from './with-mutations';
+import { concatOp, FlatteningOperator } from '../flattening-operator';
+import { Mutation, MutationResult, MutationStatus } from './mutation';
 
 export type Func<P, R> = (params: P) => R;
 
```

Comment:
I can't add comment to `RxMutationOptions` below, but it would again full names for the types.

I think we briefly chatted about a possible confusion when using both parameters `operation` and `operator`. Could we come up with a better name for `operator`? `flattener`, `flatteningOperator`, etc...